src/Logging/Logging/ScopedLogService.cs

```diff
@@ -38,46 +38,69 @@ public bool IsEnabledFor(LogLevel logLevel)
         public virtual void DisableLogging() => 
             IsEnabled = false;
 
+
         public void Log<TState>(
             DateTimeOffset timestamp,
             LogLevel logLevel,
             TState state,
-            Exception ex,
-            Func<TState, Exception, string> formatter)
+            string methodName = "")
         {
-            Argument.NotNull(formatter, nameof(formatter));
-
-            var message = CreateMessage(timestamp,
-                logLevel,
-                state,
-                ex,
-                formatter);
+            Log(timestamp, logLevel, state, null, null, methodName);
+        }
 
-            PublishMessage(message);
+        public void Log<TState>(
+            LogLevel logLevel,
+            TState state,
+            string methodName = "")
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)